inference-engine/src/gna_plugin/gna_plugin.cpp

```diff
@@ -717,7 +717,7 @@ void GNAPlugin::LoadNetwork(CNNNetwork & _network) {
 
     //  Check the input network
     std::string error;
-    if (!AreLayersSupported(network, error)) {
+    if (!GNALimitations::Cnn2D::Validator::AreLayersSupported(network, error)) {
```

Comment:
Why have you moved common function to 2d validator?

Reply:
done